Homework_Arrays/Task3_2.java

```diff
@@ -0,0 +1,37 @@
+package Homework_Arrays;
+
+import java.util.Scanner;
+
+public class Task3_2 {
+
+    public static void main(String[] args) {
+        //Scanner scanner = new Scanner(System.in);
+
+        //System.out.print("Enter the number of elements in the array: ");
+        //int n = scanner.nextInt();
+
+       // int[] arr = new int[n];
+
+        //System.out.println("Enter elements of the array:");
+
+        int[] arr = {1, 2, 3, 4, 5, 6};
+        int length = arr.length;
+        int [] updated = new int[length];
+
+        for (int i = 0; i < length/2; i++) {
```

Comment:
In your solution the output is: 4, 5, 6, 0, 0, 0
actually should be: 4, 5, 6, 6, 5, 4